tests/test_invalid_actions.py

```diff
@@ -191,6 +192,24 @@ def test_callback(tmp_path):
     model.learn(100, callback=MaskableEvalCallback(Monitor(eval_env), eval_freq=100, warn=False))
 
 
+def test_child_callback():
+    """
+    Stop callback and callback on new best rewards
+    """
+
+    env = make_env()
+    eval_env = make_env()
+    model = MaskablePPO("MlpPolicy", env)
+    stop_callback = StopTrainingOnNoModelImprovement(1, 2)
+    new_best_mean_callback = EventCallback()
+    eval_callback = MaskableEvalCallback(Monitor(eval_env), eval_freq=1, callback_after_eval=stop_callback,
+                                         callback_on_new_best=new_best_mean_callback)
+    model.learn(1000, callback=eval_callback)
```

Comment:
```suggestion
    model.learn(128, callback=eval_callback)
```

tests should run as fast as possible